### PR TITLE
the one where badges in a links_list with images works

### PR DESCRIPTION
**Pull request by sturobson**
Screenshot 2020-05-18 at 10 41 10

### CODE REVIEW

**Comment by sturobson** [comment]
might need an overhaul so it looks nicer if there's lots of text and you're on a mobile device.

I think the only way we can do that is with a breaking change in the markup.

Screenshot 2020-05-18 at 11 01 38

File: components/vf-banner/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (khawkins98) [inline components/vf-banner/package.json:2]: Are the package.json files accidental commits? They might mess with lerna?

**Review by khawkins98 — APPROVED** [review]
Aside from my worry on the package.json files, looks good.

File: components/vf-banner/package.json (hunk 1 above)
Comment (sturobson) [reply components/vf-banner/package.json:2]: I seemed to have published it to NPM before it got merged 

https://www.npmjs.com/package/@visual-framework/vf-banner

**Pull request merged**